x/gamm/keeper/msg_server.go

```diff
@@ -192,6 +193,9 @@ func (server msgServer) SwapExactAmountIn(goCtx context.Context, msg *types.MsgS
 			sdk.EventTypeMessage,
 			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
 			sdk.NewAttribute(sdk.AttributeKeySender, msg.Sender),
+			sdk.NewAttribute(types.AttributeKeyTokensIn, msg.TokenIn.String()),
+			sdk.NewAttribute(types.AttributeKeyTokensOut, tokenOutAmount.String()),
+			sdk.NewAttribute(types.AttributeTakerFee, takerFee.String()),
```

Comment:
isn't `takerFee` is just the param value? 
I guess Tzachi needs the **amount** of taker fee.

also not sure why
`EventTypeMessage = "message"`
if we refactore it anyway, we probably can have dedicated eventtype for each tx

Reply:
yeah, I thought it was a bit weird.